### PR TITLE
Adds a 1D forcing testcase with BGC (ISPOL)

### PR DESCRIPTION
**Pull request by njeffery**
Atmospheric, ocean and biogeochemical data from the 2004 Ice Station Polarstern (ISPOL) Weddell Sea (67.9S, 54W) experiment from June 16, 2004 to December 31, 2004. Core 2 from the ISPOL location fills out the remainder of the year.

A uniform_1D initial condition was added for flexible initialization. A new testcase - single_cell_ispol - was added to
MPAS-Dev/MPAS-Seaice_standalone_framework to verify this forcing and initializtion. https://github.com/MPAS-Dev/MPAS-Seaice_standalone_framework/pull/15

BFB

### CODE REVIEW

**Comment by rljacob** [comment]
@akturner and @eclare108213 please review.

**Comment by eclare108213** [comment]
Darin and I spent several hours yesterday trying to get me up and running on anvil, and failed.  Adrian's testing setup and even E3SM itself fail to run (E3SM segfaults).  So please don't hold up merging PRs, waiting on reviews from me. I'll keep trying, but E3SM has been failing for me ever since it was called ACME.

File: components/mpas-seaice/src/Registry.xml
Comment (akturner) [inline components/mpas-seaice/src/Registry.xml:3802]: I think this implementation could be greatly simplified if the input forcing files included a nCells=1 dimension so that an airTemperature1D variable etc in addition to the airTemperature variable would not be needed

File: components/mpas-seaice/src/shared/mpas_seaice_initialize.F
Comment (akturner) [inline components/mpas-seaice/src/shared/mpas_seaice_initialize.F:569]: Do thermodynamic tracer values not need to be set by this testcase with colpkg_init_trcr (e.g. https://github.com/E3SM-Project/E3SM/blob/master/components/mpas-seaice/src/shared/mpas_seaice_initialize.F#L705?) It seems from the IPSOL namelist file that vertical thermodynamics is active so would need the tracers initialized?

**Review by akturner — CHANGES_REQUESTED** [review]

File: components/mpas-seaice/src/shared/mpas_seaice_initialize.F
Comment (njeffery) [reply components/mpas-seaice/src/shared/mpas_seaice_initialize.F:569]: @akturner :  Yes, thanks.  I added the active tracer initializations and re-tested.

File: components/mpas-seaice/src/Registry.xml
Comment (njeffery) [reply components/mpas-seaice/src/Registry.xml:3802]: @akturner : Agreed.  I chose to make the forcing data grid independent though so I could test on an arbitrary grid.  It runs with the QU240, for example, at least in short tests.

**Comment by rljacob** [comment]
@eclare108213 please review.

**Comment by eclare108213** [comment]
The conflict needs to be fixed before I can compile this.  @njeffery what cases did you run that were BFB?  Is there a particular test (or test suite) that would be most appropriate for me to run, to make sure that something hasn't inadvertently been broken in E3SM?

**Comment by njeffery** [comment]
@eclare108213 : I ran a short 10-day with restart GCASE on chrysalis with mpassi defaults and compared mpassi restarts with master.  These were BFB.  Simulation scripts for these runs are here /home/ac.jeffery/SimulationScripts/archive/v2/ with run.ISPOLTestGCASE.sh  using the feature branch and run.ISPOLTestGCASE.Master.sh pre-feature.

File: components/mpas-seaice/src/Registry.xml
Comment (akturner) [reply components/mpas-seaice/src/Registry.xml:3802]: @njeffery If the forcing is the same everywhere on the mesh, what's the utility of being able to run on meshes other than the single cell one?

Comment (njeffery) [reply components/mpas-seaice/src/Registry.xml:3802]: @akturner : your point is well taken.  I'll restructure the test case with a cell index in the forcing files. I'd like to keep the bgc nutrient file grid independent so that I can use it more generally, i.e. in a standard GCASE but with ocean bgc off.  Does that seem reasonable?

**Comment by rljacob** [comment]
@akturner have your comments been addressed?

**Comment by njeffery** [comment]
@rljacob :  I still have some changes to make in response to @akturner's comments.

**Comment by rljacob** [comment]
@njeffery how are those changes going?

**Comment by njeffery** [comment]
I've opened a new PR  ( https://github.com/E3SM-Project/E3SM/pull/5506 ) to address the issues raised by @akturner .  This one can be closed.

**Pull request closed without merging**
